pull_request_template.md

```diff
@@ -33,7 +33,7 @@ How does the proposed change impact existing implementations? What action is nee
 
 ## PR Checklist
 <!--- Put an `x` in all the boxes that apply. -->
-- [ ] I have included a CHANGELOG entry with the PR.
+- [ ] I have followed the Conventional Commits guidelines.
```

Comment:
I also agree about not being sure 😉 But changing the checklist definitely makes it less confusing as a start.

For small PRs, I'd probably opt for "squash and merge" so that the PR commit itself contains the conventional commit.

For bigger ones, even less sure. I hate asking for big stuff like that after the fact (can you rebase your whole PR pretty plz??), but it does improve housekeeping in the long run.